test/cmd/apps.sh

```diff
@@ -510,11 +510,12 @@ run_rs_tests() {
   # Pre-condition: no replica set exists
   kube::test::get_object_assert rs "{{range.items}}{{$id_field}}:{{end}}" ''
   # Command
-  kubectl create -f hack/testdata/frontend-replicaset.yaml "${kube_flags[@]}"
+  local replicaset_name="frontend-no-cascade"
+  sed -r 's/^(\s*)(name\s*:\s*frontend\s*$)/\1name: '"${replicaset_name}"'/' hack/testdata/frontend-replicaset.yaml | kubectl create "${kube_flags[@]}" -f -
```

Comment:
Add a todo to revert this after the bug is fixed?

Reply:
Added todo